### ISSUE TITLE
adjustments field help text and requirements for alt text field

### ISSUE THREAD

**Issue by hannahackermans** [elmcip/elmcip#425]
The KB includes many media for which an ALT TEXT is required but many media do not have an alt text at the moment, making it inaccessibly to screen readers. 

Currently, the field help text for alt text is: 

> Alternative text is used by screen readers, search engines, and when the image cannot be loaded. By adding alt text you improve accessibility and search engine optimization.

I suggest to change this to this text to encourage people to fill in the field and give them suggestions for how to do it:

> Alternative text is used by screen readers, search engines, and when the image cannot be loaded. Adding alt text is required _by the EU_ to improve accessibility and search engine optimization.
General advice is to indicate the nature of the media first (i.e. screenshot, book cover, screen capture video, etc) and then shortly describe the content.

(I put _by the EU_ is in italics because I am not sure if it is the EU or a more specific organ that requires the alt text.)

I also vote for making alt text a required field, perhaps with a short explanation in the error indicating that this is anew requirement and is therefore popping up when editing records created by other people. all in all I don't think it would cost them that much extra time.
![Screenshot 2019-06-18 10 53 01](https://user-images.githubusercontent.com/18713442/59667972-f3647300-91b7-11e9-8e50-78ff5497ba53.png)

@steinmb Is the alt text one field that shows up in different places, or are they different fields in different places? Should I go through the website and collect all the alt text fields?

This issue is related to #352 which also addresses the accessibility of the Knowledge Base

**Comment by steinmb** [comment]
>@steinmb Is the alt text one field that shows up in different places, or are they different fields in different places? Should I go through the website and collect all the alt text fields?

+1 for the issue. I have to have a look at how the alt text is stored. Since ELMCIP have been through many iterations there could be currently more than one way the data is stored, though normally only one way it get rendered to the client meaning we might expose media that have ALT text stored but it still not output when we create the HTML. This issue will req. some digging to make sure we get them all though but mostly it is manual labor.

https://www.deque.com/blog/great-alt-text-introduction/ https://en.wikipedia.org/wiki/Alt_attribute

**Comment by steinmb** [comment]
We could whip up a quick view of images and then provide at link to media. That will speed up the work a lot:

## Example:

![2019-09-09 12-52-40 2019-09-09 12_54_20](https://user-images.githubusercontent.com/683617/64525451-32125100-d301-11e9-94c9-a6232091282e.gif)

**Comment by elmcip** [comment]
There's also something weird going on when you upload images. You enter the title, alt info first before you upload:

<img width="1190" alt="Screenshot 2019-09-25 14 49 13" src="https://user-images.githubusercontent.com/5831053/65602483-44001f00-dfa4-11e9-8d5c-35b35497776f.png">

But when you are back in the entry screen, the data has not come over to the relevant fields

<img width="801" alt="Screenshot 2019-09-25 14 49 55" src="https://user-images.githubusercontent.com/5831053/65602556-5da16680-dfa4-11e9-85bf-3819c62f9eb5.png">

And then there's that note about it not being picked up by all formatters.

**Comment by steinmb** [comment]
Going to re-test this.

**Comment by steinmb** [comment]
Verified that there is a UX thingy with the title and alt properties. Split @elmcip finding out in a separate issue since it is kind of unrelated to this. Postponding and moving this on issue #458
